release-team/role-handbooks/documentation-guides/update-release-docs-new.md

```diff
@@ -1,88 +1,180 @@
+
+- Add the label `do-not-merge/hold` to the PR.
+
+2. Add a tracking comment to the megabranch PR.
+
+Use this placeholder text:
+
+```
+## Release X.Y megabranch
+
+PR | Based on `release-X.Y`? | Tech LGTM | Docs approved | Merged
+---|---|---|---|---
+| | | |
+```
+
+    A tracking comment helps track the identity and status of component PRs in the megabranch.
+
+    For example: https://github.com/kubernetes/website/pull/8046#issuecomment-380226165
+
+2. Update the Netlify _vnext staging_ site to use the new branch:
+
+- Login to [Netlify](https://app.netlify.com/) and navigate to the Sites tab.
+
+- Open the settings for the [kubernetes-io-vnext-staging](https://app.netlify.com/sites/kubernetes-io-vnext-staging/settings) site.
+
+- Edit the Deploy settings and change "Branch" to the new version branch, for example: `release-1.11`.
+
+- Save the change and verify that the change is live at [https://kubernetes-io-vnext-staging.netlify.com/](https://kubernetes-io-vnext-staging.netlify.com/).
+
+- Update the [pull request template](https://github.com/kubernetes/kubernetes.github.io/blob/master/.github/PULL_REQUEST_TEMPLATE.md) with the version in development. For example: https://github.com/kubernetes/website/pull/8057
 
-*   Generate reference docs and place them in kubernetes/website. See [PR 6366](https://github.com/kubernetes/website/pull/6366).
-*   Update reference page for feature gates. See [PR 6364](https://github.com/kubernetes/website/pull/6364).
+## Continue the release cycle
+
+### Track PRs
+
+- Follow the feature spreadsheet.
+
+    Keep the feature tracking spreadsheet up to date with review progress and merge status for each documentation PR. For example: [Kubernetes Features OSS tracking board (1.11 release)](https://docs.google.com/spreadsheets/d/16N9KSlxWwxUA2gV6jvuW9N8tPRHzNhu1-RYY4Y0RZLs/edit#gid=0)
+
+- Update documentation PR progress in the megabranch PR's tracking comment. For example, [Release 1.11 (tracking comment)](https://github.com/kubernetes/website/pull/8046#issuecomment-380226165)
+
+- Make sure that docs PRs for the release have the correct base and set the correct Milestone. For example, feature PRs for version 1.11 need a base branch of `release-1.11` and the Milestone set to `1.11`.
+
+### Maintain the current and upcoming release branches
+
+- Periodically merge `master` into the current release branch.
+
+For example, during the v1.11 release cycle, you must periodically merge `master` into `release-1.10`.
+
+Periodic merges must continue throughout the release cycle, until the pending release (for example, `release-1.11`) merges to `master`. This is to ensure that the current release branch remains up to date, and to preserve the current branch as a snapshot when the new version is released. Periodic maintenance also helps avoid excessive merge conflicts between `master` and the current branch at the end of the release cycle.
```

Comment:
you mean we do this until the code thaw period ? i.e., as long as PRs from master is accepted into the release?